packages/gamut/src/Form/SelectDropdown.tsx

```diff
@@ -104,6 +105,7 @@ export const SelectDropdown: React.FC<SelectDropdownProps> = ({
   placeholder = 'Select an option',
   inputProps,
   isSearchable = false,
+  shownOptions,
```

Comment:
Can replace this https://github.com/Codecademy/client-modules/pull/1957/files#diff-fcaeb7ab98af88e2ffbd7eee363c51f3a0b9e77ae092caf53c94037e16a8c424R169

```suggestion
  shownOptions = 6,
```